worksheet_C/README.md

```diff
@@ -0,0 +1,19 @@
+a) This algorithm checks if there is a duplicate in the list. It does this by starting with the first item in the list and checking, in order, every other item in the list to see if they match. It then does this with the second item. When it finds a duplicate it returns true. 
+
+B)  Because there are is a for loop within a for loop (a nested loop). This means in the worst case the algorithm will have to check through n*n times to find the duplicate.  
+
+C) It is still correct because every value of j is still checked with every value of I.  
```

Comment:
Not quite true -- for example i=1 and j=2 will never be checked, however element 1 will be checked against element 2. (I think this is what you mean, but referring to i and j here makes it unclear.)